mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -301,7 +355,7 @@ def ncr(n, r):
             for iteration, (c, cv_scores) in work:
 
                 self.subsets_[iteration] = {
-                    "feature_idx": c,
+                    "feature_idx": tuple(c),
```

Comment:
Maybe, return a tuple in function `_merge_lsts(...)` so that we do not change this line here. It might be cleaner